src/game/engine/gscreen.h

```diff
@@ -114,7 +114,7 @@ class GameScreenClass
     virtual void Render();
     virtual void Draw_It(BOOL force_redraw);
     virtual void Blit_Display();
-    virtual void Set_Default_Mouse(MouseType mouse, BOOL a2 = false) = 0;
+    virtual void Set_Default_Mouse(MouseType mouse, BOOL in_radar = false) = 0;
     virtual BOOL Override_Mouse_Shape(MouseType mouse, BOOL a2 = false) = 0;
```

Comment:
The argument for Override_Mouse_Shape() looks like it is the same as Set_Default_Mouse().